config/rapidez/sentry.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+return [
+    // Configuration as defined in the docs: https://docs.sentry.io/platforms/javascript/guides/vue/configuration/
+    'configuration' => [
+        'enabled' => true,
+
+        // Amount of errors to be logged to sentry (1.00 = 100%)
+        'sampleRate' => 1.00,
```

Comment:
This is de recommended value?

Reply:
More specifically, this is the default. Rather keep defaults like this as the default :)

Reply:
L11 set the precedent that everything is configurable via .env, maybe we could follow suit (with defaults of course)

Reply:
We definitely could, but that does mean that we would probably have to pre-define a ton of configurations in here.

Could also go the magento route and dynamically retrieve any relevant env settings without putting them in the config file?

Reply:
So long as the configurations we place are configurable it will be fine, any other additions in the future can be done via PRs or overriding the config. To support all options by default is a large task, while most might not even get used